MSBLOC.Web/Services/JsonWebTokenService.cs

```diff
@@ -22,6 +22,7 @@ public JsonWebTokenService(IOptions<AuthOptions> optionsAccessor)
             _optionsAccessor = optionsAccessor;
         }
 
+        /// <inheritdoc cref="IJsonWebTokenService.CreateToken"/>
```

Comment:
1. `cref="IJsonWebTokenService.CreateToken"` is redundant here.
2. `inheritdoc` does not actually do anything, unless you provide docs at the source, `IJsonWebTokenService.CreateToken`